### PR TITLE
LPS-162318 - Check Layout related to CPT has privateLayout column set…

### PR DESCRIPTION
**Pull request by younes2123**
Motivation
=======
[LPS-162318](https://liferay.atlassian.net/browse/LPS-162318) : Poshi Automation for Bug [LPS-156013](https://liferay.atlassian.net/browse/LPS-156013)

Steps to Verify
========
Suggestions for testing: add this to your test.me,properties, change password by yours.

`   #
    # MySQL 
    #

    database.mysql.device.name=
    #database.mysql.docker.image=
    #database.mysql.docker.run.options=
    database.mysql.driver=com.mysql.cj.jdbc.Driver
    database.mysql.host=localhost
    database.mysql.password=
    database.mysql.port=3306
    database.mysql.schema=lportal
    database.mysql.service.cmd.start=mysqld start
    database.mysql.service.cmd.stop=mysqld stop
    database.mysql.service.executable=service
    database.mysql.username=root
    database.mysql.url=jdbc:mysql://${database.mysql.host}/${database.mysql.schema}?characterEncoding=UTF-8&dontTrackOpenResources=true&holdResultsOpenOverStatementClose=true&permitMysqlScheme&serverTimezone=GMT&useFastDateParsing=false&useUnicode=true
    database.mysql.version=8.0.22
    database.mysql.version.cmd=--version`

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+definition {
+
+	property ci.retries.disabled = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Content Pages";
+
+	static var randomSiteName = StringUtil.randomString(8);
+
+	static var randomSiteURLKey = StringUtil.toLowerCase(${randomSiteName});
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:11]: Do not use random name anymore

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 2]
```diff
@@ -0,0 +1,85 @@
+@component-name = "portal-wcm"
+definition {
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:3]: Add properties for remote staging
```
		property app.server.bundles.size = "1";
		property databases.size = "1";
		property minimum.slave.ram = "24";
	property custom.properties = "auth.verifier.TunnelAuthVerifier.hosts.allowed=${line.separator}tunneling.servlet.shared.secret=1234567890123456";
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+				remoteStaging = "true",
+				siteId = ${siteId},
+				siteName = ${randomSiteName});
+		}
+	}
+
+	@description = "Check if Layout related to content page templates are published as private layout in live and staging"
+	@priority = 4
+	test LayoutRelatedToCPTArePublishedAsPrivateLayoutInLiveAndStaging {
+		task ("Given content page template is created") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:42]: Add properties
```
		property test.liferay.virtual.instance = "false";
		property test.run.type = "single";
		```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+				pageTemplateName = "Content Page Template Name",
+				pageTemplateType = "Content Page Template");
+		}
+
+		task ("And fragment is added to content page template") {
+			PageTemplatesAdmin.openPageTemplatesAdmin(siteURLKey = ${randomSiteURLKey});
+
+			Navigator.gotoNavNested(navNested = "Page Template Collection");
+
+			PageTemplatesAdmin.gotoPageTemplate(pageTemplateName = "Content Page Template Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:57]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+		}
+
+		task ("Activate remote staging") {
+			Staging.openStagingAdmin(siteURLKey = ${randomSiteURLKey});
+
+			Staging.activateStagingCP(
+				remotePort = ${remotePort},
+				remoteStaging = "true",
+				siteId = ${siteId},
+				siteName = ${randomSiteName});
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:35]:
```
			Staging.remoteStagingSetUp(
				remotePort = 9080,
				siteName = "Test Site Name");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+		task ("Set up virtual instance") {
+			TestCase.setUpPortalInstance();
+		}
+
+		task ("Sign in") {
+			User.firstLoginPG();
+		}
+
+		task ("Add a site") {
+			HeadlessSite.addSite(siteName = ${randomSiteName});
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:25]: ```HeadlessSite.addSite(siteName = "Test Site Name");```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+				remotePort = ${remotePort},
+				remoteStaging = "true",
+				siteId = ${siteId},
+				siteName = ${randomSiteName});
+		}
+	}
+
+	@description = "Check if Layout related to content page templates are published as private layout in live and staging"
+	@priority = 4
+	test LayoutRelatedToCPTArePublishedAsPrivateLayoutInLiveAndStaging {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:41]: Move testcase file to `pagetemplate/contentpagetemplate/` then rename to ViewContentPageTemplateAsPrivateLayoutInStagingAndLive

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+			Staging.activateStagingCP(
+				remotePort = ${remotePort},
+				remoteStaging = "true",
+				siteId = ${siteId},
+				siteName = ${randomSiteName});
+		}
+	}
+
+	@description = "Check if Layout related to content page templates are published as private layout in live and staging"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:39]: This is a test for LPS-156013. Check if Layout related to content page templates are published as private layout in live and staging.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase (hunk 1 above)
Comment (younes2123) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPageTemplateWithRemoteStaging.testcase:11]: Thank you, I will update the PR right now

**Comment by younes2123** [comment]
@Tim-Cao Can you please check the changes, thanks.

**Comment by Tim-Cao** [comment]
@younes2123, see changes https://github.com/liferay-echo/liferay-portal/pull/14584

**Pull request closed without merging**